src/mobile/Footer/Footer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './Footer.scss';
+
+interface Props{
+  footerLinks: React.ReactElement;
+  footerInfo: React.ReactElement;
+}
+
+export function Footer({footerLinks, footerInfo}: Props) {
```

Comment:
Props will never change. Why do you need them? Why not to use needed components in `Footer` component? And it's harder to read code if you pass components as props.